include/openssl/asn1t.h.in

```diff
@@ -558,63 +607,14 @@ struct ASN1_ITEM_st {
                                      * the contents */
     long tcount;                /* Number of templates if SEQUENCE or CHOICE */
     const void *funcs;          /* functions that handle this type */
+    /*
+     * TODO rename 'funcs' to make it less misleading and make clear in its type
+     * that it can be ASN1_PRIMITIVE_FUNCS*, ASN1_EXTERN_FUNCS*, or ASN1_AUX*
```

Comment:
This is public API so the TODO is kind of - some time in the distant future when we decide to do an API break of this kind.
So maybe just leave a description of what kinds of pointers might be there instead of marking it as TODO would be more apropriate.

Reply:
Good point - done.